lua/entities/gmod_wire_expression2/core/angle.lua

```diff
@@ -173,17 +175,26 @@ end
 
 __e2setcost(2)
 
-// SET methods that returns angles
+-- SET methods that returns angles
 e2function angle angle:setPitch(rv2)
-	return Angle(rv2, this[2], this[3])
+	local ang = Angle(this)
+	ang.pitch = rv2
+
+	return ang
```

Comment:
Are you certain this is an improvement?

Reply:
The benchmark showed that it works a little faster.